src/attpc_engine/detector/transporter.py

```diff
@@ -141,11 +144,9 @@ def point_transport(
         (x,y) position of point being transported.
     electrons: int
         Number of electrons made at point being transported.
-
-    Returns
-    -------
-    numpy.ndarray
-        An Nx3 array representing the simplified point cloud
+    points: dict[int, int]
+        A dictionary mapping a unique pad,tb key to the number of electrons, which
+        will be filled by this function
     """
     # Find pad number of hit pad, if it exists
     point = np.full((1, 3), -1.0)
```

Comment:
Acutally this was just a straight up bug (not really bug but def misleading) by me. So we don't want to return points from point transport, the function should modify the dictionary it was given in the function call and place any valid point in the dictionary, otherwise just do nothing. The reasoning is that this makes the API of using point_transport vs transverse_transport the same. I've fixed this and everything should be good now.

Reply:
Ah sorry I just read your comment more carefully and yeah you already said everything I just said above. Ugh reading comprehension is not strong today hahaha.